pytest-shutil/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pytest-shutil" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2cfe4d3f5f25ad2b19e64847d62563f5494b2e0450ca1cfc5940974029b2cbd1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - contextlib2
+    - execnet
+    - mock
+    - path.py
+    - pip
+    - pytest
+    - python
+    - setuptools
+    - six
+    - termcolor
+  run:
+    - contextlib2
+    - execnet
+    - mock
+    - path.py
+    - pytest
+    - python
+    - setuptools
+    - six
+    - termcolor
+
+test:
+  imports:
+    - pytest_shutil
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/manahl/pytest-plugins
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Please change that to
```
  license_file: 'LICENSE'
```
The license file is already being downloaded, so, it's not necessary to upload that as well